tools/pip/setup.py

```diff
@@ -152,9 +152,12 @@ def skip_markdown_comments(md):
     if os.path.exists(os.path.join(libdir, 'libgfortran.so.3')):
         shutil.copy(os.path.join(libdir, 'libgfortran.so.3'), mxdir)
         package_data['mxnet'].append('mxnet/libgfortran.so.3')
-    else:
+    elif os.path.exists(os.path.join(libdir, 'libgfortran.so.4')):
         shutil.copy(os.path.join(libdir, 'libgfortran.so.4'), mxdir)
         package_data['mxnet'].append('mxnet/libgfortran.so.4')
+    else:
+        shutil.copy(os.path.join(libdir, 'libgfortran.so.5'), mxdir)
+        package_data['mxnet'].append('mxnet/libgfortran.so.5')
```

Comment:
it's possible that user compiled and statically linked libgfortran or this file isn't necessary, so I'd recommend to add the condition check.

Reply:
The `tools/pip/` is currently coupled with the `tools/staticbuild`. We should uncouple them but I think it can be part of a separate PR.